VikingQuiz.UI/src/Components/MyQuizesPage/MyQuizesPage.css

```diff
@@ -0,0 +1,31 @@
+.row{
+    margin-right:0;
```

Comment:
Please format the code. There should be a space between the column ( ':' ) and the value for the property.